app/javascript/src/Dashboard/styles/dashboard.scss

```diff
@@ -0,0 +1,43 @@
+$font-primary: #464646;
+
+.ApiFilter {
+
+  float: left;
+  width: auto;
+  border: 1px solid #d1d1d1;
+  border-radius: 4px;
+  margin-top: -0.5rem;
+  margin-bottom: 0.5rem;
+
+  span {
+    margin: 0.5rem;
+    color: $font-primary
+  }
+
+  input {
+    border: none;
+    width: auto;
+    padding: 0.5rem;
+  }
+
+  ::-webkit-input-placeholder {
+    font-style: normal;
+    font-weight: 600;
+    color: $font-primary;
+  }
+  ::-moz-placeholder {
```

Comment:
placeholder styles are defined in https://github.com/3scale/porta/blob/dashboard-links/app/assets/stylesheets/provider/_forms.scss#L481